### DISCUSSION TITLE
How do I install cookie consent on Wordpress?

### DISCUSSION THREAD

**Discussion by yohanbey** [orestbida/cookieconsent#739] [Q&A]
Hello,

I would like to be able to set up Cookie Consent on a Wordpress site instead of using a plugin,

I'd like to be able to set up Cookie Consent on a Wordpress site instead of using a plugin and I know it's possible because I've seen it on a Wordpress site but I don't know how to do it.

If anyone can help me, thank you.

**Comment by lombrone** [comment]
@yohanbey hey, check here for wordpress install: https://github.com/orestbida/cookieconsent/discussions/754
ignore the `script_loader_tag` part, and enqueue style too

**Comment by codewithzac** [comment]
I wrote a thing for this over the Christmas break - it seems to work well for the websites I'm using it on, but very open to additional feedback if you'd like to give it a go: https://github.com/codewithzac/wordpress-cookieconsent-loader
